### PR TITLE
feat(audit): introducing carbon design system audit

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
#5940
https://app.zenhub.com/workspaces/carbon-for-ibmcom-5d449f3642eb1962336cbe52/issues/carbon-design-system/carbon-for-ibm-dotcom/5940

### Description
This PR adds the Carbon Design System bucket that adds the Color Token Audit and the Text Token Audits. 

### Changelog

**New**

- Color Token audit and Text Token audit

### CODE REVIEW

File: src/config/custom-config.js [hunk 1]
```diff
@@ -97,6 +104,10 @@ module.exports = {
       title: UIStrings.carbonForIBMDotcomTitle,
       description: str_(UIStrings.carbonForIBMDotcomDescription),
     },
+    'carbon-design-system': {
+      title: 'Carbon Design System',
+      description: 'Test', //TODO
```
Comment (jeffchew) [inline src/config/custom-config.js:109]: Could we get an actual description here?

**Comment by jeffchew** [comment]
More of a question, is there a way to list the actual issues found in the report? 
image

Also, might be worth linking to the dev tools so they can inspect manually, but not sure if you can do more than one link here?

**Comment by IgnacioBecerra** [comment]
@jeffchew I've added a table listing out the tokens with the wrong values along with the ones that should be being used 👍

**Comment by jeffchew** [comment]
@IgnacioBecerra thank you, looks great!

image

**Review by jeffchew — APPROVED** [review]
LGTM!

**Pull request merged**